Sources/PipecatClientIOS/types/rest/ApiRequest.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public struct APIRequest: Codable {
+    public let endpoint: URL
+
+    /// Custom HTTP headers to be sent with the POST request to baseUrl.
+    public let headers: [[String: String]]
+
+    /// Custom parameters to add to the auth request body.
+    public let requestData: Value?
+
+    public let timeout: TimeInterval?
```

Comment:
it looks like maybe the `timeout` value isn't used to bail if the request takes too long. Is that not needed on iOS? if not, maybe don't include it here.